staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -640,10 +640,12 @@ func (s preparedGenericAPIServer) NonBlockingRun(stopCh <-chan struct{}, shutdow
 	}()
 
 	s.RunPostStartHooks(stopCh)
-
-	if _, err := systemd.SdNotify(true, "READY=1\n"); err != nil {
-		klog.Errorf("Unable to send systemd daemon successful start message: %v\n", err)
-	}
+  
+	go func (){
+		if _, err := daemon.SdNotify(false, "READY=1\n"); err != nil {
+			klog.Errorf("Unable to send systemd daemon successful start message: %v\n", err)
+		}
+	}()
```

Comment:
Here you convert synchronous call into async (goroutine) call. It seems harmless in this particular case, but still.